### ISSUE TITLE
File could not be opened as resource archive

### ISSUE THREAD

**Issue by DARKKin09** [morkt/GARbro#485]
I am trying to unpack .rio files (Muv Luv Photonmelodies) but every time I try to I just get the error 'File could not be opened as resource archive'. 
I've tried doing what [#20](https://github.com/morkt/GARbro/issues/20) did, but to no avail. I have no idea what I could be doing wrong here.

I'm running Windows 10 x64 
GARbro version v1.5.44.2904

**Comment by tsudoko** [comment]
rUGP support in various extraction tools is notoriously bad. GARbro can only open rio files from very old games such as 君がいた季節 (1999); there is some code to support obfuscated archives used in newer titles but I couldn't get it to open anything. You might have better luck with [RioX](http://riox.thisillusion.org/) (or [this patched version](https://forums.fuwanovel.net/topic/23674-muv-luv-photonmelodies-and-photonflowers-h-scene-restoration/?do=findComment&comment=525886)).

**Comment by DARKKin09** [comment]
Yeah from what I've learned while researching the subject rUGP seems like a dumpster fire to get through. Thank you for your response, the link to the patched version seems dead and I've already tried the standard RioX version. 

It is what it is I suppose. I am however curious about TDA assets, would you happen to know anything related to getting those using some way? (Sorry if this is getting too off topic)
